.gitignore

```diff
@@ -1,5 +1,4 @@
 .idea
-/system/wymeditor/
```

Comment:
This commit needs a better commit message too. It does not really add WYMEditor, since we were already using that. It does add the *source* or WYMEditor, but also makes a lot of other changes to how WYMEditor works. Stating these changes explicitly would be better. Also, please mention the version of WYMEditor that is added, for future reference (version number and/or git hash), and whether any changes were made to the upstream wymeditor code (if so, adding the code verbatim first and then modifying the code in a separate commit would be preferred).

Reply:
The commit is split as suggested, the commit message is updated.